### ISSUE TITLE
Cannot find module 'ajv'

### ISSUE THREAD

**Issue by geoffreywiseman** [codekie/openapi-examples-validator#18]
```
$ npm install -g openapi-examples-validator
/usr/local/bin/openapi-examples-validator -> /usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js
+ openapi-examples-validator@1.1.1
added 19 packages from 19 contributors in 1.978s

$ openapi-examples-validator docs/api/submit-api.yml
internal/modules/cjs/loader.js:716
    throw err;
    ^

Error: Cannot find module 'ajv'
Require stack:
- /usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:713:15)
    at Function.Module._load (internal/modules/cjs/loader.js:618:27)
    at Module.require (internal/modules/cjs/loader.js:771:19)
    at require (internal/modules/cjs/helpers.js:68:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:9482)
    at a (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:124)
    at Object.<anonymous> (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:8745)
    at a (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:124)
    at Object.<anonymous> (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:5233)
    at a (/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js:2:124) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js'
  ]
}
```

On NPM, ajv is listed as a dev dependency, which is presumably why it's not installed, but it seems to be required at runtime?

**Comment by geoffreywiseman** [comment]
Similarly:
```
$ npx openapi-examples-validator
Cannot find module 'ajv'
Require stack:
- /usr/local/lib/node_modules/openapi-examples-validator/dist/cli.js
```

**Comment by codekie** [comment]
Thanks for the report. `ajv` is not supposed to be a dev-dependency.

**Comment by codekie** [comment]
fixed in 1.1.2
